src/core/Akka/Akka.csproj

```diff
@@ -53,7 +53,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
+    <DefineConstants>$(DefineConstants);CORECLR;AKKAIO</DefineConstants>
```

Comment:
No, not yet. It would have to be the next thing to fix/port. MultiNodeTestRunner wouldn't work without NodeTestRunner, and NodeTestRunner requires Akka.IO